diff --git a/node_modules/react-aws-s3-typescript/dist/Url.js b/node_modules/react-aws-s3-typescript/dist/Url.js
index 29b41b4..2f29341 100644
--- a/node_modules/react-aws-s3-typescript/dist/Url.js
+++ b/node_modules/react-aws-s3-typescript/dist/Url.js
@@ -7,7 +7,7 @@ var buildUrl = function (_a) {
         case 'cn':
             return "https://" + bucketName + ".s3." + region + ".amazonaws.com." + countryCode;
         default:
-            return "https://" + bucketName + ".s3-" + region + ".amazonaws.com";
+            return "https://" + bucketName + ".s3." + region + ".amazonaws.com";
     }
 };
 exports.default = (function (config) {
